src/fetchAllData.js

```diff
@@ -0,0 +1,41 @@
+function fetchData() {
+  let userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
+    .then(response => response.json())
+    .then(data => {
+      return data.userData;
+    })
+    .catch(err => console.log(err.message))
```

Comment:
We will want to do something more with these errors later but good catch thus far.

Reply:
I think console logging is a bit cleaner!